server/plugins/IOUGens.cpp

```diff
@@ -1165,7 +1209,7 @@ FLATTEN void XOut_next_a_nova(XOut* unit, int inNumSamples) {
 
     if (xfade0 != next_xfade) {
         float slope = CALCSLOPE(next_xfade, xfade0);
-        for (int i = 0; i < numChannels; ++i) {
+        for (int i = 0; i < numChannels; ++i, out += bufLength) {
```

Comment:
Is there a reason `out` wasn't incremented before, or was this a bug?

Reply:
As the commit message indicates ("XOut: fix broken NOVA_SIMD version") this was a bug.